library/build.gradle

```diff
@@ -46,15 +46,15 @@ android {
 
 ext {
     guavaVersion = "29.0-android"
-    junitVersion = "4.13"
-    mockitoVersion = "3.5.7"
+    junitVersion = "4.13.1"
+    mockitoVersion = "3.5.11"
```

Comment:
Actually, the upgrade of other libraries should come with a different PR, as this PR aims to migrate to Androidx.  Let's leave it as this time, luckily all tests were passed on CI and also locally :)